### PR TITLE
Lukasp/make ci green again

### PR DESCRIPTION
**Pull request by LukasPukenis**

### CODE REVIEW

File: drop-storage/src/lib.rs [hunk 1]
```diff
@@ -24,9 +24,10 @@ pub struct Storage {
 impl Storage {
     pub async fn new(logger: Logger, path: &str) -> Result<Self> {
         let options = SqliteConnectOptions::from_str(path)?.create_if_missing(true);
-        let conn = SqlitePool::connect_with(options).await?;
+        let conn = SqlitePool::connect_with(options.clone()).await?;
+
         sqlx::migrate!("./migrations")
-            .run(&conn)
+            .run(&mut SqlitePool::connect_with(options).await?.acquire().await?)
```
Comment (matszczygiel) [inline drop-storage/src/lib.rs:30]: Why?

File: drop-storage/src/lib.rs [hunk 2]
```diff
@@ -49,7 +50,7 @@ impl Storage {
             "INSERT OR IGNORE INTO peers (id) VALUES (?1)",
             transfer.peer
         )
-        .execute(&mut conn)
+        .execute(conn.acquire().await?)
```
Comment (matszczygiel) [inline drop-storage/src/lib.rs:53]: It should probably be `&mut *conn`

File: drop-storage/src/lib.rs [hunk 3]
```diff
@@ -109,7 +116,7 @@ impl Storage {
             "INSERT INTO transfer_active_states (transfer_id) VALUES (?1)",
             transfer_id,
         )
-        .execute(&mut conn)
+        .execute(conn.acquire().await?)
```
Comment (matszczygiel) [inline drop-storage/src/lib.rs:119]: Same here. It's probably not necessary

File: drop-storage/src/lib.rs (hunk 2 above)
Comment (LukasPukenis) [reply drop-storage/src/lib.rs:53]: It shouldn't compile if it would need a dereference?

Comment (LukasPukenis) [reply drop-storage/src/lib.rs:53]: Oh, I get it, it's probably that referencing allows me to omit `acquire()` part?

Comment (LukasPukenis) [reply drop-storage/src/lib.rs]: So I've tried it and yep `&mut *conn` works as well and looks a bit nicer so changed all occurences

File: drop-transfer/src/error.rs [hunk 4]
```diff
@@ -116,6 +116,6 @@ impl From<walkdir::Error> for Error {
         value
             .into_io_error()
             .map(Into::into)
-            .unwrap_or(Error::BadPath("Filesystem loop detected".into()))
+            .unwrap_or_else(|| Error::BadPath("Filesystem loop detected".into()))
```
Comment (matszczygiel) [inline drop-transfer/src/error.rs:119]: :+1:

**Review by matszczygiel — APPROVED** [review]

**Pull request merged**